core/Azure.Mcp.Core/src/Commands/CommandFactory.cs

```diff
@@ -23,7 +24,6 @@ public class CommandFactory
     private readonly ModelsJsonContext _srcGenWithOptions;
 
     private const string RootCommandGroupName = "azmcp";
```

Comment:
Other than this const, is there a reason to implement CommandFactory per server?